### DISCUSSION TITLE
Dialog and Loading Screen

### DISCUSSION THREAD

**Discussion by erthium** [Duck-Tech-Dev/app-winnerduck#10] [Ideas]
Do not bother with the texts written, but what do you think about:
- Usage of Loading Screen, having it on the bottom-right 
- Usage of dialogs from Tremor, pop-ups showing what we want to say

I found it better rather than having a simple texts saying the id is not valid, but not sure, what do you guys think @Utkub24 @HuseyinSimsek7904 ?

**General Screen:**

![screenshot-2024-03-23_01:46:20](https://github.com/Duck-Tech-Dev/app-winnerduck/assets/49292808/3b045dbf-3540-4f1f-b7be-c9693e884ef3)

---

**Loading Screen:**

![screenshot-2024-03-23_01:47:22](https://github.com/Duck-Tech-Dev/app-winnerduck/assets/49292808/0a20894e-bd94-4a9a-bde1-8c09f931596d)

---

**Pop-up Dialog**

![screenshot-2024-03-23_01:47:34](https://github.com/Duck-Tech-Dev/app-winnerduck/assets/49292808/4048e7c8-dd55-400e-afde-231e38718e32)

**Comment by Utkub24** [comment]
Could be better if the loading screen was more centered and attention grabbing. But looks nice 👌

**Comment by HuseyinSimsek7904** [comment]
I think they look okay, though like @Utkub24, the loading screen may be more attention grabbing.
